scripts/uberenv/packages/axom/package.py

```diff
@@ -254,10 +250,6 @@ def hostconfig(self, spec, prefix):
         if "+cpp14" in spec:
             cfg.write(cmake_cache_entry("BLT_CXX_STD", "c++14", ""))
 
-        # BEGIN SERAC EDIT
```

Comment:
How does this still work if you remove our edit?

Reply:
because it doesnt...

Reply:
To my credit.. the tests are still running =)

Reply:
😄 It's been a long week, and it's only Wednesday!